frontend/webservice/vrf.cgi

```diff
@@ -923,6 +987,18 @@ sub remove_vrf {
     $vrf->decom(user_id => $user->user_id);
     $db->commit;
 
+    my $error = OESS::DB::VRF::add_vrf_history(
+        db => $db,
+        event => 'decom',
+        vrf => $vrf,
+        user_id => $user->user_id,
+        workgroup_id => $wg,
+        state => 'decom'
+    );
+    if (defined $error) {
+        warn $error;
+    }
+
```

Comment:
This should be called prior to `$db->commit` to be run within the same transaction